### ISSUE TITLE
Can you search in the tab title too?

### ISSUE THREAD

**Issue by MM-John** [Quicksaver/FindBar-Tweak#260]
Often the title text is in the page. But for my roundcube email, eg, the string "Webmail" is in the title and not on the page and FindBar didnt find it.

The other usage (even bigger) is re-load firefox and unloaded pages have titles but no page content. Very important for me to be able to find these before loading the page.

**Comment by Quicksaver** [comment]
I've been planning to add this feature for a while (and I still might). But it's worth mentioning my other add-on [Tab Groups](https://addons.mozilla.org/firefox/addon/tab-groups-panorama/) can do this very easily.

**Comment by MM-John** [comment]
That would be great!

As for Tab Groups, I have a question. Bear with me. This is a subtle distinction involving very similar words and concepts: I believe the method in the tab groups addon is to replace the "window" concept with the "tab group" concept. Ie, you will only ever have 1 Windows 7 window open at a time, and instead of several "windows" each of which has tabs, you'll have instead several "tab groups" each of which has tabs. And so back to searching: tab groups will search across tab groups in the 1 window. But if you did happen to have more than 1 Windows 7 window open, then those windows would each have their own--separate--groups of tabs. So, in a way, if you replace windows with tab groups, the tab groups addon can search across tab groups. But it can't actually search across Windows 7 windows.

I dont know why but I feel uncomfortable abandoning the idea of Windows 7 windows. One thing maybe--sometimes I like to have two web page up side by side, which requires 2 separate Windows 7 windows.

**Comment by Quicksaver** [comment]
> if you replace windows with tab groups, the tab groups addon can search across tab groups. But it can't actually search across Windows 7 windows.

Exactly. I am planning to add the ability to search (and share groups) across windows, but for now that's exactly as it happens. So I imagine that's not the best solution for you.

As I said, I still plan to add the ability to search tab titles in FBT. :)
